Submitted By: Ryan Oliver <ryan.oliver AT pha.com.au>
Date: 2004-10-28
Initial Package Version: 1.2.7
Origin: Ryan Oliver
Description: Patch ensures generated pkgconfig file points to the correct
  location for the installed libraries
diff -uNr libpng-1.2.7-orig/scripts/libpng.pc.in libpng-1.2.7/scripts/libpng.pc.in
--- libpng-1.2.7-orig/scripts/libpng.pc.in	2004-09-12 12:19:36.000000000 +1000
+++ libpng-1.2.7/scripts/libpng.pc.in	2004-10-28 00:10:52.124127368 +1000
@@ -1,7 +1,7 @@
 
 prefix=@PREFIX@
 exec_prefix=${prefix}
-libdir=${exec_prefix}/lib
+libdir=@LIBPATH@
 includedir=${exec_prefix}/include
 
 Name: libpng12
diff -uNr libpng-1.2.7-orig/scripts/makefile.linux libpng-1.2.7/scripts/makefile.linux
--- libpng-1.2.7-orig/scripts/makefile.linux	2004-09-12 12:19:36.000000000 +1000
+++ libpng-1.2.7/scripts/makefile.linux	2004-10-28 00:10:24.220369384 +1000
@@ -78,7 +78,9 @@
 	$(RANLIB) $@
 
 libpng.pc:
-	cat scripts/libpng.pc.in | sed -e s\!@PREFIX@!$(prefix)! > libpng.pc
+	cat scripts/libpng.pc.in | \
+	sed -e s\!@PREFIX@!$(prefix)! \
+	    -e s\!@LIBPATH@!$(LIBPATH)! > libpng.pc
 
 libpng-config:
 	( cat scripts/libpng-config-head.in; \
